### ISSUE TITLE
handle layers appearing at different zoom levels

### ISSUE THREAD

**Issue by mikelmaron** [stevage/vector-inspector#3]
first of all, just discovered vector-inspector today, great tool

I'm working with a tileset that has layers appearing at different zoom levels (https://api.dev.climatetrace.org/v0/tiles/assets/z/x/y/) There's no TileJSON.

Depending on what tile entered, only the layers at that zoom level are available for inspection.

Perhaps there's a way to re-inspect the available layers as different zoom levels are encountered.

**Comment by bazmap** [comment]
I agree with @mikelmaron and to be more precise, it could be nice to allow overzoom in the tool : displaying data outside of the zoom bound.
Some client allow this, you need to have been through level where you can see the data to have a local copy of them but if you zoom more than the bound, the client continu to render the local data.
